src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -181,7 +175,8 @@ private LibraryTab(BibDatabaseContext bibDatabaseContext,
                        BibEntryTypesManager entryTypesManager,
                        CountingUndoManager undoManager,
                        ClipBoardManager clipBoardManager,
-                       TaskExecutor taskExecutor) {
+                       TaskExecutor taskExecutor,
+                       boolean isDummyContext) {
```

Comment:
Okay for private Constructor and factory use. Im thinking if using a null context would be more meaningful here. However, this can be discussed in a follow-up pr.